test/integration/production/test/index.test.js

```diff
@@ -56,6 +57,7 @@ describe('Production Usage', () => {
     console.log(output)
```

Comment:
Should we move the above `console.log` to an `else` branch below to prevent thinking it is expected to log twice in the event of a failure? Also does `console` need to be imported for the workaround to work properly or do we just need to use `console.error` so the output goes directly to stderr?